From 9a72d50dec7520d1d88551cba4e3aeaf3e606070 Mon Sep 17 00:00:00 2001
From: LexManos <LexManos@gmail.com>
Date: Fri, 8 Mar 2013 20:31:37 -0800
Subject: [PATCH] Early define CrashReport classes to combat invalid crash details.

---
 .../net/minecraftforge/common/MinecraftForge.java  |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/src/minecraft/net/minecraftforge/common/MinecraftForge.java b/src/minecraft/net/minecraftforge/common/MinecraftForge.java
index a98de3b..b07c957 100644
--- a/src/minecraft/net/minecraftforge/common/MinecraftForge.java
+++ b/src/minecraft/net/minecraftforge/common/MinecraftForge.java
@@ -13,6 +13,7 @@ import cpw.mods.fml.relauncher.SideOnly;
 import net.minecraft.block.Block;
 import net.minecraft.block.material.Material;
 import net.minecraft.client.renderer.texture.IconRegister;
+import net.minecraft.crash.CrashReport;
 import net.minecraft.entity.monster.EntityEnderman;
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
@@ -207,6 +208,9 @@ public class MinecraftForge
 
        EVENT_BUS.register(INTERNAL_HANDLER);
        OreDictionary.getOreName(0);
+
+       //Force these classes to be defined, Should prevent derp error hiding.
+       new CrashReport("ThisIsFake", new Exception("Not real"));
    }
 
    public static String getBrandingVersion()
-- 
1.7.3.1

